app/src/main/java/com/taskruler/MainActivity.kt

```diff
@@ -121,9 +121,7 @@ fun UserTasksList(
         Button(onClick = {
             selectedUserTask.apply {
                 activityName = inActivityName
-                activityId = selectedActivity?.let {
-                    it.activityId
-                } ?: 0
+                activityId = selectedActivity?.activityId ?: 0
```

Comment:
Redundant 'let' call
https://kotlinlang.org/docs/scope-functions.html#context-object-this-or-it

Reply:
Either way works.  
{} is a shave more readable, and allows for future expansion of the function body.